src/pages/VoterLeaderboard/index.tsx

```diff
@@ -0,0 +1,62 @@
+/** @jsxImportSource @emotion/react */
+import React, { useState } from 'react';
+import { Pagination } from 'components';
+import { useGetVoterAccounts } from 'clients/api';
+import { VoterAccount } from 'types';
+import LeaderboardTable from './LeaderboardTable';
+import { useStyles } from './styles';
+
+interface IVoterLeaderboardProps {
+  voterAccounts: VoterAccount[];
+  offset: number;
+  total: number | undefined;
+  limit: number | undefined;
```

Comment:
nit: I think making these optional would fit the spec better, since we're basically saying a `total` and a `limit` don't have to be passed rather than saying they have to be passed but can be `undefined`.

Reply:
They do need to be passed though. They are required for this component in order for pagination to work correctly. What I am trying to convey here is that they are required by could be undefined until the query loads.

Reply:
Ok I see. Works for me then!